canonical/ubuntu.com
# /download/risc-v redesign

## Done

- Applied redesign as per [mock-up](https://www.figma.com/file/WsMLgH1TXcU1mZ7YjqZRDY/u.com%2Fdownload%2Frisc-v?type=design&node-id=1-479&t=CjoQFe70UKFQNHyE-0)
- Added functionality for small screen dropdown select

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12870.demos.haus/download/risc-v
- Check against design
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3875
